app/scripts/directives/overview/virtualMachineRow.js

```diff
@@ -24,6 +24,76 @@
     templateUrl: 'views/overview/_virtual-machine-row.html'
   });
 
+  var RDP_PORT = 3389;
+
+  function fileDownload(data, fileName, mimeType) {
+    fileName = fileName || "vm.rdp";
+    mimeType = mimeType || "application/rdp";
+
+    var a = document.createElement('a');
+    if (navigator.msSaveBlob) { // IE10
```

Comment:
IE 10 is not supported

Reply:
ok. Still keeping the msSaveBlob here but removing the later branch for non-HTML5 browsers